content/influxdb/v2.0/reference/upgrading/influxd-upgrade-guide.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Upgrade from InfluxDB 1.x to InfluxDB 2.0
+description: >
+  Upgrade from InfluxDB 1.x to InfluxDB 2.0.
+menu:
+  influxdb_2_0_ref:
+    parent: Upgrade InfluxDB
+    name: Upgrade from InfluxDB 1.x to InfluxDB 2.0
+weight: 9
+---
+
+Use the `influxd upgrade` command to upgrade InfluxDB 1.x to InfluxDB 2.0.
+This command copies all data stored in [databases](/influxdb/v1.8/concepts/glossary/#database) and 
```

Comment:
1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp) to 2.0 [buckets](/influxdb/v2.0/reference/glossary/#bucket)